torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1284,11 +1321,14 @@ TensorExprKernel::BackendType TensorExprKernel::inferBackendTypeFromDevice(
   BackendType backendType = BackendType::kUninitialized;
   if (device.type() == at::kCUDA) {
     backendType = kCudaCodeGen;
+  } else if (device.type() == at::kCPU && getTEGenerateBlockCode()) {
+    backendType = kBlockCodeGen;
   } else if (device.type() == at::kCPU) {
 #ifdef TORCH_ENABLE_LLVM
     backendType = kLLVMCodeGen;
 #else
     backendType = kSimpleIREval;
+    std::cout << "device also used\n";
```

Comment:
Leftover?

Reply:
Argh! forgot this. Thanks for catching it, will remove.